sphinx_source/bezeyiciler.rst

```diff
@@ -0,0 +1,146 @@
+.. meta::
+   :description: Python 3.x'te bezeyiciler (Decoratorlar)
+   :keywords: python, fonksiyonlar, decoratorlar, bezeyiciler
+.. highlight:: py3
+
+Fonksiyonlarda Bezeyicilerin (Decorator) Kullanımı
+**************************************************
+
+Bu bölüme kadar fonksiyonları nasıl kullanabileceğimizi işlemiştik. 
+Bu yazıda decoratorların nasıl çalıştığını, decoratorların ne olduğunu ve
```

Comment:
'Decorator' yerine 'bezeyici' kullanılmalı; metnin geri kalanı için de geçerlidir.